Utils/Dataflow/pyDKB/dataflow/stage/AbstractStage.py

```diff
@@ -124,9 +124,9 @@ def defaultArguments(self):
                           )
         self.add_argument('-e', '--end-of-message', action='store', type=str,
                           help=u'custom end of message marker\n'
-                          'NOTE: in (f)ile mode for JSON messages EOM '
-                          'can be set to empty string to read input '
-                          'file as single JSON object, not as NDJSON. '
+                          'NOTE: for (f)ile sources EOM can be set to empty '
+                          'string to read input file as single message, '
```

Comment:
"as a single message", I suppose :)

Reply:
Maybe ) Sorry, somehow I missed this comment before merging the PR...( Will make as a separate PR, I think...

Reply:
Done in `pyDKB-minor-fix`; hopefully won\`t be lost.